crates/lib3h/src/engine/real_engine.rs

```diff
@@ -437,8 +442,23 @@ impl<T: Transport, D: Dht, SecBuf: Buffer, Crypto: CryptoSystem> RealEngine<T, D
                 match maybe_space {
                     Err(res) => outbox.push(res),
                     Ok(space_gateway) => {
-                        for (entry_address, _) in msg.address_map {
-                            space_gateway.post_dht(DhtCommand::HoldEntryAddress(entry_address))?;
+                        for (entry_address, aspect_address_list) in msg.address_map {
+                            let mut aspect_list = Vec::new();
+                            for aspect_address in aspect_address_list {
+                                let fake_aspect = EntryAspectData {
+                                    aspect_address: aspect_address.clone(),
+                                    type_hint: String::new(),
+                                    aspect: vec![],
+                                    publish_ts: 0,
+                                };
+                                aspect_list.push(fake_aspect);
+                            }
+                            let fake_entry = EntryData {
+                                entry_address: entry_address.clone(),
+                                aspect_list,
+                            };
```

Comment:
why are they fake? we're not in test code here, are we?

Reply:
That entryData is fake in the sense there is no actual content, but all the addresses are 'real'.
Didn't want to bother with adding a new struct for just addresses at this point.
What better term would you prefer?

Reply:
I get it. Ok, cool, yeah maybe just add a comment in here, and I'm fine with fake :+1:

Reply:
👍